chain/cosmos/chain_node.go

```diff
@@ -653,7 +653,7 @@ func (tn *ChainNode) CreateNodeContainer() error {
 }
 
 func (tn *ChainNode) StopContainer() error {
-	return tn.Pool.Client.StopContainer(tn.Container.ID, uint(time.Second*30))
```

Comment:
Documentation of the 3rd party package says this unit of time is already seconds. 

Not the best package API. I prefer `time.Duration` for time arguments.